web-common/src/features/dashboards/time-series/totals-data-store.ts

```diff
@@ -27,12 +29,28 @@ export function createTotalsForMeasure(
       useTimeControlStore(ctx),
       ctx.dashboardStore,
     ],
-    ([runtime, metricsViewName, timeControls, dashboard], set) =>
-      createQueryServiceMetricsViewAggregation(
+    ([runtime, metricsViewName, timeControls, dashboard], set) => {
+      const measuresList: V1MetricsViewAggregationMeasure[] = measures.flatMap(
+        (measureName) => {
+          const baseMeasure = { name: measureName };
+          if (!includeComparison) {
+            return [baseMeasure];
+          }
+          return [
+            baseMeasure,
+            {
+              name: measureName + ComparisonDeltaPreviousSuffix,
+              comparisonValue: { measure: measureName },
+            },
+          ];
+        },
+      );
+
+      return createQueryServiceMetricsViewAggregation(
```

Comment:
Note for the future: this should be moved to a stable query observer using the pattern:
```ts
const queryOptionsStore = derived (...)
return createQuery(queryOptionsStore)
```